### PR TITLE
cnf-tests: fix dpdk vhost test

### PR DESCRIPTION
**Pull request by SchSeba**
This commit give some time to the second dpdk application(client) to start and send traffic

Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

File: cnf-tests/testsuites/e2esuite/dpdk/dpdk.go [hunk 1]
```diff
@@ -271,13 +277,15 @@ sleep INF
...
 				By("Parsing output from the DPDK application")
-				Eventually(func() string {
+				Eventually(func() bool {
 					out, err = pods.GetLog(dpdkWorkloadPod)
 					Expect(err).ToNot(HaveOccurred())
-					return out
-				}, 8*time.Minute, 1*time.Second).Should(ContainSubstring("NIC statistics for port"),
-					"Cannot find accumulated statistics")
-				checkRxOnly(out)
+					if !strings.Contains(out, "NIC statistics for port") {
```
Comment (fedepaol) [inline cnf-tests/testsuites/e2esuite/dpdk/dpdk.go:283]: is this necessary? Without it, checkRxOnly would perform the same check I guess

Comment (SchSeba) [reply cnf-tests/testsuites/e2esuite/dpdk/dpdk.go]: you are right thanks! :)

**Pull request merged**